MetadataExtractor/Formats/Exif/makernotes/PanasonicMakernoteDescriptor.cs

```diff
@@ -492,7 +696,7 @@ public string GetConversionLensDescription()
         [CanBeNull]
         public string GetDetectedFacesDescription()
         {
-            return BuildFacesDescription(Directory.GetDetectedFaces());
+            return BuildFacesDescription(Directory.GetDetectedFaces()) ?? "0";
```

Comment:
Does this tag always have a value in this descriptor? Elsewhere in the library we return `null` when the tag is not present. This change would suggest no faces were detected if the method were called directly, even though perhaps the tag didn't have a value. Consider changing the implementation of `BuildFacesDescription`, but it should still return `null` if the tag was omitted.